pkg/cmd/cli/secrets/remove_secret_from_object.go

```diff
@@ -0,0 +1,114 @@
+package secrets
+
+import (
+	"fmt"
+	"io"
+
+	kapi "k8s.io/kubernetes/pkg/api"
+	kcmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	RemoveSecretRecommendedName = "remove"
+
+	// TODO: move to examples
+	removeSecretLong = `
+Remove (detach) secrets from a ServiceAccount
+If a secret is no longer valid for a pod, build or image pull, you may remove it from a service account.`
+
+	removeSecretExample = ` // Remove a secret currently associated with a service account like this:
+%[1]s serviceaccount/sa-name secret/secret-name secret/another-secret-name ...`
```

Comment:
I have that because it's a direct mirror of the `oc secrets add` format. They require it there, so `remove` does too. Consistency is more important than brevity, IMHO.